### ISSUE TITLE
Sign in session storage (cookie)?

### ISSUE THREAD

**Issue by desrogers** [derekgeorg/project2#62]
User has to sign in again if they go to the landing page.

I'm not sure what we should do about this, if we should do anything. It's kind of a minor detail. But something I noticed

**Comment by urangel** [comment]
Good catch. I think I know how to solve this. I think we check if req.user exists within the route for the landing page and if it exists then don't let them go there again.
